src/api/curriculumApi.ts

```diff
@@ -71,22 +140,88 @@ export async function fetchCompetenceGoal(
   const json: CompetenceGoal = await resolveJson(response);
   return {
     id: json.id,
```

Comment:
Id på kompetansemålet er egentlig ikkje interessant for oss. Det er bedre at vi setter inn kode her slik at begge kompetansemåltypene har id på ca samme format. Då treng vi heller ikkje code-feltet lenger nede.
```suggestion
    id: json.kode,
```